packages/documentation/src/stories/components/subnavigation/subnavigation.docs.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta, Source, Canvas } from '@storybook/blocks';
+import StylesPackageImport from '../../../shared/styles-package-import.mdx';
+import * as subnavigationStories from './subnavigation.stories';
+
+<Meta of={subnavigationStories} />
+
+# Subnavigation
+
+<p className="lead">This component elegantly organizes navigation links into intuitive horizontal tabs, 
```

Comment:
Was it written with ChatGPT? 😁
More seriously, we usually use action verbs for the lead. I would suggest something like: "Supplement the header navigation with a secondary entry-point, providing access to additional features in your application."

Reply:
Yes, this was a suggestion from ChatGPT :) Your suggestion sounds good, I will use this.